src/page-content/content-mfa.tsx

```diff
@@ -0,0 +1,191 @@
+import React from 'react'
+import Button from '../components/freestanding/button/button'
+import {PropTypes as FeatureContentProps} from '../components/layouts/feature-content/feature-content'
+import {PropTypes as FeaturelistProps} from '../components/layouts/featurelist/feature-list'
+import {PropTypes as HeroImageProps} from '../components/layouts/hero/hero-image'
+import {PropTypes as SeoProps} from '../components/layouts/seo/seo'
+import CLIExample from '../images/asciicasts/ory-cli-example.svg'
+import Automation from '../images/illustrations/standards.svg'
+
+const CheckCircle = (
+    <i className={'ph-check-circle-fill themed-primary size16'}/>
+)
+const Keyhole = <i className={'ph-keyhole-fill size16'}/>
+const Browser = <i className={'ph-browser-fill size16'}/>
+const ArrowRight = <i className={'ph-arrow-right-bold size16'}/>
+export const seo: SeoProps = {
+    description:
+        'Add Multi-factor Authentication to guard your users privacy. Learn how Ory MFA secures the verification process.',
+    title: 'Ory Multi-Factor Authentication (MFA)'
+}
+export const hero: HeroImageProps = {
+    id: 'cli.hero',
+    title: (
+        <>
+            Introducing rigorous security with{' '}
```

Comment:
```suggestion
            Introduce rigorous security with{' '}
```


I think its always better to use the simple form in heros and CTAs, it directly speaks to the reader (introduce this to make X)